amclient/amclientargs.py

```diff
@@ -132,6 +133,27 @@
     default="amclient-transfer",
     type=None,
 )
+MICROSERVICE_GROUP = Opt(
+    name="microservice-group",
+    metavar="MICROSERVICEGROUP",
+    help="Filter by microservice group",
+    default=None,
+    type=None,
+)
+MICROSERVICE_LINK_UUID = Opt(
+    name="microservice-link-uuid",
```

Comment:
Just an aside as we iron these API endpoints out but getting the consumer a list of these does seem to be something we need. Although they'll always be able to find a link uuid out by calling the endpoint once... One we can consider in docs maybe?

Reply:
We still don't have a use case for `link_uuid`s. Maybe it's worth removing for now?

Reply:
I don't think they do any harm and they fit in well with the internationalization aspect of this work. A UUID will always be consistent where the labels might change from language to language, but also between changes to `workflow.json`.

Reply:
:+1: